src/views/LikeDislike/Dislike.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { dislikeArticle } from "../../redux/actions/likeDislike";
+import Button from "../Button";
+
+class Dislike extends React.Component {
+  handleClick = () => {
+    const { dispatch } = this.props;
+    dispatch(dislikeArticle());
+    const element = document.getElementById("dislikeButton");
+    element.classList.add("bounceIn");
+    window.setTimeout(() => {
+      element.classList.remove("bounceIn");
+    }, 1000);
+  };
+
+  render() {
+    const { disliked } = this.props;
+    return (
+      <div>
+        <Button
+          className="btn btn-warning"
+          id="dislikeButton"
+          role="button"
+          tabIndex="-1"
+          onClick={this.handleClick}
+          onKeyDown={this.handleClick}
+          label="Dislike"
+          disabled={disliked ? "disabled" : ""}
+        />
+        {/* <i
+          id="dislikeButton"
```

Comment:
What is the purpose of this commented code?

Reply:
These were files that I was not using. They have since been deleted